inc/template-functions.php

```diff
@@ -101,7 +102,8 @@ function wmf_get_role_hierarchy( $parent_id ) {
 	);
 
 	foreach ( $terms as $term_id => $parent ) {
-		if ( 0 < $parent ) {
+		// Ensure that $parent is an integer and greater than 0.
+		if ( is_int( $parent ) && $parent > 0 ) {
```

Comment:
I think we can actually drop this comment, it describes the line of code exactly which feels duplicative. I don't believe this logic would be confusing without the comment.